content/_index.md

```diff
@@ -39,6 +39,6 @@ src="./images/taiwan-unsplash-2x.jpg"
 alt="Sunset over Taipei City"
 width="793"
 height="428"
-desktopOnly=true >}}
+desktopOnly="true" >}}
```

Comment:
I have no idea what I'm doing in go land, I wanted to just pass the "desktopOnly" option as a boolean but couldn't work out how. This works but we don't really need the string "true" in there.

Is there a better way?